packages/sit-onyx/src/i18n/locales/de-DE.json

```
@@ -38,5 +77,8 @@
     "goBack": "Zurück",
     "userMenuLabel": "Benutzeroptionen"
   },
-  "showInfoTooltip": "Info-Tooltip anzeigen"
+  "showTooltip": {
+    "info": "Info-Tooltip anzeigen",
+    "error": "Error-Tooltip anzeigen"
```

```suggestion
    "error": "Fehler-Tooltip anzeigen"
```

Is "Error" a German word?

---

so much denglish, I forgot how to german 🤦🏻‍♀️ I felt sure it is but nope.. ^^'